plugin/role_set.go

```diff
@@ -51,12 +53,26 @@ func (rs *RoleSet) validate() error {
 		err = multierror.Append(err, fmt.Errorf("role set should have account associated"))
 	}
 
-	if len(rs.Bindings) == 0 {
-		err = multierror.Append(err, fmt.Errorf("role set bindings cannot be empty"))
-	}
+	if rs.UseStaticServiceAccount {
+		if rs.AccountId.EmailOrId == "" || rs.AccountId.Project == "" {
+			err = multierror.Append(err, fmt.Errorf("role set should have service account & project associated"))
```

Comment:
I think it's better to add `when using static service account` on this error message too.

Reply:
This has been resolved in 544f1acc9f20c5a5cc1cee5b5ce47eb0141828d4